dotnetv3/Glue/scenarios/Glue_Basics_Scenario/Glue_Basics/GlueMethods.cs

```diff
@@ -0,0 +1,496 @@
+        /// information.</param>
+        /// <returns>A Boolean value indicating whether information about
+        /// the AWS Glue job runs was retrieved successfully.</returns>
+        public static async Task<bool> GetJobRunsAsync(AmazonGlueClient glueClient, string jobName)
+        {
+            var runsRequest = new GetJobRunsRequest
+            {
+                JobName = jobName,
+                MaxResults = 20,
+            };
+
+            var response = await glueClient.GetJobRunsAsync(runsRequest);
+            var jobRuns = response.JobRuns;
+
+            if (jobRuns.Count > 0)
+            {
+                foreach (JobRun jobRun in jobRuns)
+                {
+                    Console.WriteLine($"Job run state is {jobRun.JobRunState}");
+                    Console.WriteLine($"Job run Id is {jobRun.Id}");
+                    Console.WriteLine($"The Glue version is {jobRun.GlueVersion}");
+                }
+
+                return true;
+            }
+            else
+            {
+                Console.WriteLine("No jobs found.");
+                return false;
+            }
+        }
+
+        // snippet-end:[Glue.dotnetv3.GlueBasicsScenario.GetJobRunsAsync]
+
+        // snippet-start:[Glue.dotnetv3.GlueBasicsScenario.GetSpecificCrawlerAsync]
+
+        /// <summary>
+        /// Retrieves information about a specific AWS Glue crawler.
+        /// </summary>
+        /// <param name="glueClient">The initialized AWS Glue client.</param>
+        /// <param name="crawlerName">The name of the crawer.</param>
+        /// <returns>A Boolean value indicating whether information about
+        /// the AWS Glue crawler was retrieved successfully.</returns>
+        public static async Task<bool> GetSpecificCrawlerAsync(AmazonGlueClient glueClient, string crawlerName)
+        {
+            var crawlerRequest = new GetCrawlerRequest
+            {
+                Name = crawlerName,
+            };
+
+            var response = await glueClient.GetCrawlerAsync(crawlerRequest);
+            if (response.HttpStatusCode == System.Net.HttpStatusCode.OK)
+            {
+                var databaseName = response.Crawler.DatabaseName;
+                Console.WriteLine($"{crawlerName} has the database {databaseName}");
+                return true;
+            }
+
+            Console.WriteLine($"No information regarding {crawlerName} could be found.");
+            return false;
+        }
+
+        // snippet-end:[Glue.dotnetv3.GlueBasicsScenario.GetSpecificCrawlerAsync]
+
+        // snippet-start:[Glue.dotnetv3.GlueBasicsScenario.GetSpecificDatabaseAsync]
+
+        /// <summary>
+        /// Gets information about the database created for this Glue
+        /// example.
+        /// </summary>
+        /// <param name="glueClient">The initialized Glue client.</param>
+        /// <param name="databaseName">The  name of the AWS Glue database.</param>
```

Comment:
Remove extra space between "The" and "name"

Reply:
fixed